src/router/index.ts

```
@@ -13,4 +14,18 @@ const router = createRouter({
 
   }
 })
+
+// reroute when address is undefined
+router.beforeEach((to, from, next) => {
+  const store = useWalletStore();
+  if (to.meta && to.meta.noAuth) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)